src/Microsoft.DotNet.Cli.Utils/Constants.cs

```diff
@@ -20,6 +20,7 @@ internal static class Constants
 
         public static readonly string HostExecutableName = "corehost" + ExeSuffix;
         public static readonly string DefaultConfiguration = "Debug";
+        public static readonly string DefaultPublishConfiguration = "Release";
```

Comment:
Agreed with @brthor, it should be consistent. Changes like this shouldn't be made in a PR that refactors code. It needs to be agreed upon before implementing.